octobot_evaluators/evaluators/realtime_evaluator.py

```diff
@@ -53,6 +54,8 @@ def _get_tentacle_registration_topic(self, all_symbols_by_crypto_currencies, tim
             if ideal_time_frame != to_handle_time_frame:
                 self.logger.warning(f"Missing {ideal_time_frame.name} time frame in available time frames, "
                                     f"using {to_handle_time_frame.name} instead.")
+            # set self.available_time_frame with the actually available one
+            self.available_time_frame = to_handle_time_frame.value
```

Comment:
Why not setting `self.time_frame`?

Reply:
I did it at first but `self.time_frame` is read from config, I think it's better not to erase it for clarity

Reply:
ok :+1: